cms/static/js/spec/video/translations_editor_spec.js

```diff
@@ -1,8 +1,8 @@
 define(
     [
-        'jquery', 'underscore', 'common/js/spec_helpers/ajax_helpers', 'squire'
+        'jquery', 'underscore', 'squire'
     ],
-function ($, _, AjaxHelpers, Squire) {
+function ($, _, Squire) {
```

Comment:
Same question: is this just removing an unused import, or did it cause test failures?